doc/api/spec.rst

```diff
@@ -954,6 +954,32 @@ for example, it will store the random forest with ``n_estimators=5``, and
 uses square brackets to differentiate them from regular placeholders when
 using an ``env.yaml`` file.
 
+In the latest version, we have introduced a special flag called `grid_number_suffix`.
```

Comment:
please add a specific version here (this would be 0.21.1) and add an entry to the changelog under this section, see this: https://github.com/ploomber/ploomber/blob/master/CONTRIBUTING.md#documenting-changes-and-new-features

Reply:
Added version and updated docstring.